src/features/animation/share-utils.ts

```
@@ -114,7 +120,13 @@ export const generatePlatformUrl = (
       return url;
     case "notion":
       // Notion requires the embed URL format
-      return `https://www.notion.so/embed?url=${encodedUrl}&title=${encodedTitle}`;
+      const notionUrl = `https://www.notion.so/embed?url=${encodedUrl}&title=${encodedTitle}`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)